### PR TITLE
Russian misspelling correction

### PR DESCRIPTION
**Pull request by Riliane**
No related ticket as it's a simple fix

I am a new community member and this is my first pull request! I am looking forward to contributing further.

### Submitter checklist

- [n/a] (If applicable) Jira issue is well described
- [n/a] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [n/a] Appropriate autotests or explanation to why this change has no tests
- [n/a] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 
- [n/a] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@Pldi23 @yaroslavafenkin

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Review by Wadeck — APPROVED** [review]
Welcome to the community and thanks for your first contribution 🎉 

For other reviewers, the only actual change is about a removal of a whitespace. The lowercase to uppercase should have no impact.

**Review by rsandell — APPROVED** [review]

File: core/src/main/resources/jenkins/model/Jenkins/_safeRestart_ru.properties [hunk 1]
```diff
@@ -1,3 +1,3 @@
...
+Are\ you\ sure\ about\ restarting\ Jenkins?\ Jenkins\ will\ restart\ once\ all\ running\ jobs\ are\ finished.=\u0412\u044B \u0443\u0432\u0435\u0440\u0435\u043D\u044B \u043D\u0430\u0441\u0447\u0435\u0442 \u043F\u0435\u0440\u0435\u0437\u0430\u0433\u0440\u0443\u0437\u043A\u0438 Jenkins? Jenkins \u0431\u0443\u0434\u0435\u0442 \u043F\u0435\u0440\u0435\u0437\u0430\u0433\u0440\u0443\u0436\u0435\u043D, \u043A\u043E\u0433\u0434\u0430 \u0432\u0441\u0435 \u0437\u0430\u043F\u0443\u0449\u0435\u043D\u043D\u044B\u0435 \u0437\u0430\u0434\u0430\u0447\u0438 \u0437\u0430\u0432\u0435\u0440\u0448\u0430\u0442\u0441\u044F.
```
Comment (Pldi23) [inline core/src/main/resources/jenkins/model/Jenkins/_safeRestart_ru.properties:1]: nit: For me it not sounds naturally, maybe better to translate: `Вы уверены что хотите перезагрузить Jenkins? Jenkins будет перезагружен, когда все запущенные задачи завершатся.` ?

**Review by Pldi23 — APPROVED** [review]
Welcome to Jenkins community!

File: core/src/main/resources/jenkins/model/Jenkins/_safeRestart_ru.properties (hunk 1 above)
Comment (Riliane) [reply core/src/main/resources/jenkins/model/Jenkins/_safeRestart_ru.properties:1]: I have checked and that's actually how it is translated in _restart.ru.properties (except with a comma). It's indeed a good idea to match it up!

Comment (Pldi23) [reply core/src/main/resources/jenkins/model/Jenkins/_safeRestart_ru.properties:1]: Yep, with comma sentence would be grammatically correct. `Вы уверены, что хотите перезагрузить Jenkins?`

**Review by yaroslavafenkin — APPROVED** [review]

**Review by Pldi23 — APPROVED** [review]

**Comment by MarkEWaite** [comment]
This PR is now ready for merge.  We will merge it after approximately 24 hours if there is no negative feedback.

**Pull request merged**